### DISCUSSION TITLE
how do I change language in slicer

### DISCUSSION THREAD

**Discussion by artem56881** [OrcaSlicer/OrcaSlicer#3638] [Q&A]
Hello, I don't live in an English-speaking country, but I'd like my slicer to be in English. Couldn't find any settings or config files. Please help.

**Comment by LJKraus** [comment]
Use Ctrl-P on the key board, or In the top left corner click the Down arrow to select "Preferences".   Probably looks different in the currently set language .
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/263647ee-1989-43c3-9d01-1747f9640bc0)
